### DISCUSSION TITLE
Testdata values range

### DISCUSSION THREAD

**Discussion by ianmoone** [grafana/grafana#37015] [Data Sources]
**What would you like to be added**:
When using TestData datasource, it would be really useful to be able to specify a range with min and max values to set limits in the values which are generated.
**Why is this needed**:
This would add some kind of real world touch when you want to assert the usability of your dashboards with the different teams involved.

**Comment by torkelo** [comment]
yea, good idea. Been wanting this myself

**Comment by FireEmerald** [comment]
At least for the **Random Walk** Scenario, min and max values can now be set (Grafana 7.0.6).
![2020-07-22 10_20_42-Window](https://user-images.githubusercontent.com/1107939/88152975-1bb57100-cc05-11ea-867a-fea00e6f430a.png)

It would be cool to have these options also for other Scenarios, like **Predictable Pulse** or **Predictable CSV Wave**.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
